packages/svelte/src/compiler/phases/2-analyze/index.js

```diff
@@ -540,7 +542,8 @@ export function analyze_component(root, source, options) {
 		source,
 		snippet_renderers: new Map(),
 		snippets: new Set(),
-		async_deriveds: new Set()
+		async_deriveds: new Set(),
+		hoisted_promises: new Map()
```

Comment:
maybe we rename this to `promise_aliases` to disambiguate with the other `hoisted_promises`?